cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -54,22 +56,8 @@ func CreateLocalEtcdStaticPodManifestFile(manifestDir, patchesDir string, nodeNa
 	if cfg.Etcd.External != nil {
 		return errors.New("etcd static pod manifest cannot be generated for cluster using external etcd")
 	}
-	// gets etcd StaticPodSpec
-	spec := GetEtcdPodSpec(cfg, endpoint, nodeName, []etcdutil.Member{})
 
-	// if patchesDir is defined, patch the static Pod manifest
-	if patchesDir != "" {
-		patchedSpec, err := staticpodutil.PatchStaticPod(&spec, patchesDir, os.Stdout)
-		if err != nil {
-			return errors.Wrapf(err, "failed to patch static Pod manifest file for %q", kubeadmconstants.Etcd)
-		}
-		spec = *patchedSpec
-	}
-
-	// writes etcd StaticPod to disk
-	if err := staticpodutil.WriteStaticPodToDisk(kubeadmconstants.Etcd, manifestDir, spec); err != nil {
-		return err
-	}
+	getAndWriteEtcdStaticPod(manifestDir, patchesDir, cfg, endpoint, nodeName, []etcdutil.Member{}, isDryRun)
```

Comment:
should check for returned error here.
same for the call in CreateStackedEtcdStaticPodManifestFile.

Reply:
Good catch! done!